### DISCUSSION TITLE
Sponsor not showing up on my profile

### DISCUSSION THREAD

**Discussion by xames3** [community/community#64512] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello hello everyone!

I started my GitHub sponsorship program yesterday and it got verified. I got a sponsor too about in an hour of making my profile live. One of my followers sponsored me too but I can't see his profile on my GitHub account page as Sponsor. My previous sponsor is visible but not this one. He reached out to me and I tried to discuss with me tracing back his steps but still his name isn't showing on my profile. I can see him in my Sponsors Dashboard though. He got the Sponsorship Badge too but he can't show himself as a sponsor to my account. Any ideas why?

**Comment by arafajs** [comment]
There are a few reasons why a sponsor might not be showing up on your GitHub profile:

- The sponsor might not have enabled the "Show my sponsorship on the sponsor's profile" setting. This setting can be found in the "Sponsorship" tab of the sponsor's profile.
- The sponsor might have sponsored you through a third-party platform, such as Patreon or Liberapay. In this case, the sponsor's name will not show up on your GitHub profile.
- There might be a delay in the system. Sometimes it takes a few hours for sponsorships to show up on profiles.
- If you are sure that the sponsor has enabled the "Show my sponsorship on the sponsor's profile" setting and that they sponsored you through GitHub Sponsors, then you can contact GitHub Support for help.

**Reply by xames3** [reply]
Thanks @arafajs for a prompt response! Appreciate it!

The sponsor has enabled showing sponsorship and has been sponsoring me through GitHub sponsors program (no third-party platform). It could be the case wherein I may need to wait for some time to get his account showing up. But I'll consider raising a ticket with GitHub support team too if this is not resolved by tomorrow.

**Comment by xames3** [comment]
I guess waiting for some time solved the problem. I'm able to see the sponsor on my account profile now.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
